recipes-backports/curl/curl_8.4.0.bb

```diff
@@ -0,0 +1,139 @@
+SUMMARY = "Command line tool and library for client-side URL transfers"
```

Comment:
- is the current kirkstone recipe for curl too old?
- is this a stock recipe from a newer release?
- is it possible to drop all of that `ptest` stuff for simplicity?

Reply:
1. yes, in kirkstone it's 7.82 with a huge amount of backports for security fixes, however we need at least 7.84 because of an assumption in everest-core: https://github.com/EVerest/everest-core/blob/b0908b309bf56531613e4e582ea1a4bfa1099f15/modules/LemDCBM400600/main/http_client.hpp#L34
2. yes, from current poky master: https://git.yoctoproject.org/poky/plain/meta/recipes-support/curl/?h=master
3. sure

Reply:
alright